tests/Projects/ProjectGitSyncerTest.php

```diff
@@ -111,4 +126,19 @@ public function testCheckoutBranch(): void
 
         $this->projectGitSyncer->checkoutBranch($repositoryName, $branchName);
     }
+
+    public function testIsRepositoryInitialized(): void
+    {
+        self::assertTrue($this->projectGitSyncer->isRepositoryInitialized('orm'));
+        self::assertFalse($this->projectGitSyncer->isRepositoryInitialized('foo'));
+    }
+
+    public function testCheckoutTag(): void
+    {
+        $this->processFactory->expects(self::once())
+            ->method('run')
+            ->with('cd \'vfs://projects/example-project\' && git clean -xdf && git checkout tags/\'1.0.0\'');
```

Comment:
I would minimize the escaping here

```suggestion
            ->with("cd 'vfs://projects/example-project' && git clean -xdf && git checkout tags/'1.0.0'");
```